pandas/tests/io/json/test_pandas.py

```diff
@@ -681,10 +662,10 @@ def test_series_roundtrip_empty(self, orient, numpy):
         tm.assert_series_equal(result, expected)
 
     @pytest.mark.parametrize("numpy", [True, False])
-    def test_series_roundtrip_timeseries(self, orient, numpy):
-        data = self.ts.to_json(orient=orient)
+    def test_series_roundtrip_timeseries(self, orient, numpy, datetime_series):
```

Comment:
this looks really similar to test_series_roundtrip_empty and test_series_roundtrip_object.  would it make sense to something like `any_series` fixture for this?

Reply:
Yea I think a great idea. I’m thinking worth doing after breaking out and setting up test_roundtrip.py with other parametrization unless a blocker here

Reply:
sounds good.  This LGTM